js/plot/util/fill.js

```diff
@@ -121,6 +123,54 @@ module.exports = function(pool, opts) {
         });
 
       fills.exit().remove();
+
+      // Add midnight markers
+      if (opts.isDaily) {
+        selection.selectAll('rect.d3-fill-midnight')
+        .data(_.filter(currentData, {startsAtMidnight: true}))
+        .enter()
+        .append('rect')
+        .attr({
+          x: function(d, i) {
+            var pos;
+            if (opts.dataGutter) {
```

Comment:
I *think* this `opts.dataGutter` logic is unnecessary since we're only adding to the daily and not weekly view. If I'm right, remove for simplicity?